core/module_specifier.rs

```diff
@@ -211,9 +211,19 @@ impl PartialEq<String> for ModuleSpecifier {
 mod tests {
   use super::*;
 
+  fn get_path(specifier: &str) -> Url {
```

Comment:
Move this function definition inside test_resolve_import - otherwise looks good.

Reply:
Sure! 👍